buildkite.sh

```diff
@@ -27,7 +27,27 @@ steps:
     retry:
       automatic:
         limit: 2
+EOF
+
+case "${BUILDKITE_REPO:-x}" in
+    "https://github.com/dlang/dmd.git" | \
+    "https://github.com/dlang/druntime.git" | \
+    "https://github.com/dlang/phobos.git")
+
+cat << 'EOF'
+steps:
+  - command: |
+        make -f posix.mak style
+    label: "Style"
+EOF
+        ;;
+    *)
+        ;;
+esac
 
+################################################################################
+
+cat << 'EOF'
   - wait
```

Comment:
This will now also block on the `style` target. It looks like the only two ways around this are:
- new pipeline
- after this `wait`ing step